external/webserver/http_client.c

```diff
@@ -336,20 +367,22 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
 					}
 					len->message_len += buf_len;
 
-					response->total_len = len->content_len;
+					if(response) {
```

Comment:
Please leave spaces as per the coding rules. "if(response){" --> "if (response) {"

Reply:
done